src/main/java/redis/clients/jedis/exceptions/JedisBatchOperationException.java

```diff
@@ -0,0 +1,16 @@
+package redis.clients.jedis.exceptions;
+
+/**
+ * @deprecated This class is extending {@link JedisDataException} to maintain backward
+ * compatibility. But this will not do so in future. Instead, This would extend
+ * {@link JedisException} directly.
+ */
+@Deprecated
```

Comment:
The comment doesn't mean it's deprecated though? What's the recommendation here if it's deprecated, is it that the user should catch JedisException intead?